src/main/java/com/cglx/Calculator.java

```diff
@@ -31,9 +37,25 @@ public int square(int a)
         return a * a;
     }
 
-    private void readFile() {
-        FileInputStream fis = new FileInputStream(new File("TestFile.txt"));
-        
+    private List<String> readFile(String filename)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "readFile" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01.ahprod.adhaven.com:9990/coding_rules#rule_key=squid%3AUnusedPrivateMethod)